integration/it-integration/src/main/java/se/inera/intyg/rehabstod/integration/it/dto/PopulateFiltersResponseDTO.java

```diff
@@ -20,20 +20,19 @@
 package se.inera.intyg.rehabstod.integration.it.dto;
 
 import java.util.List;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import se.inera.intyg.infra.sjukfall.dto.DiagnosKapitel;
 import se.inera.intyg.infra.sjukfall.dto.Lakare;
 
+@Data
+@NoArgsConstructor
 public class PopulateFiltersResponseDTO {
     private List<Lakare> activeDoctors;
+    private List<DiagnosKapitel> diagnosisChapters;
 
-    public PopulateFiltersResponseDTO(List<Lakare> activeDoctors) {
-        this.activeDoctors = activeDoctors;
-    }
-
-    public List<Lakare> getActiveDoctors() {
-        return activeDoctors;
-    }
-
-    public void setActiveDoctors(List<Lakare> activeDoctors) {
+    public PopulateFiltersResponseDTO(List<Lakare> activeDoctors, List<DiagnosKapitel> diagnosisChapters) {
```

Comment:
Since we decided to use @NoArgsConstructor, maybe we can use @AllArgsConstructor to generate this constructor?